src/utils/useAtomsDevtools.ts

```diff
@@ -8,8 +8,13 @@ import { getReduxExtension } from './redux-extension/getReduxExtension';
 import { SnapshotOptions, useAtomsSnapshot } from './useAtomsSnapshot';
 import { useGotoAtomsSnapshot } from './useGotoAtomsSnapshot';
 
-const atomToPrintable = (atom: AnyAtom) =>
-  atom.debugLabel ? `${atom}:${atom.debugLabel}` : `${atom}`;
+const atomToPrintable = (atom: AnyAtom) => {
```

Comment:
We have copies of this function across the codebase, could you pls update those too or perhaps move it to a shared function [somewhere in the internals](https://github.com/jotaijs/jotai-devtools/tree/227170689eb68f006c5e60677d581f4f932f089d/src/utils/internals)?  

https://github.com/search?q=repo%3Ajotaijs%2Fjotai-devtools%20atomToPrintable&type=code

Reply:
> or perhaps move it to a shared function [somewhere in the internals](https://github.com/jotaijs/jotai-devtools/tree/227170689eb68f006c5e60677d581f4f932f089d/src/utils/internals)?

I think I like this option more.

Reply:
> atomToPrintable

> https://github.com/search?q=repo%3Ajotaijs%2Fjotai-devtools%20atomToPrintable&type=code
These are only the ones named as such. There may be more.

Reply:
Good point! There is [one more to update then](https://github.com/jotaijs/jotai-devtools/blob/227170689eb68f006c5e60677d581f4f932f089d/src/utils/useAtomDevtools.ts#L29) 😄

Reply:
I reviewed this list. I actually don't think any further changes are necessary.

1. Changed in this PR
[src/utils/useAtomsDevtools.ts](https://github.com/jotaijs/jotai-devtools/blob/227170689eb68f006c5e60677d581f4f932f089d/src/utils/useAtomsDevtools.ts#L11)
```ts
const atomToPrintable = (atom: AnyAtom) =>
  atom.debugLabel ? `${atom}:${atom.debugLabel}` : `${atom}`;
```

2. Only outputs debugLabel, so not the same logic as [src/utils/useAtomsDevtools.ts](https://github.com/jotaijs/jotai-devtools/blob/227170689eb68f006c5e60677d581f4f932f089d/src/utils/useAtomsDevtools.ts#L11)
[src/utils/useAtomsDebugValue.ts](https://github.com/jotaijs/jotai-devtools/blob/227170689eb68f006c5e60677d581f4f932f089d/src/utils/useAtomsDebugValue.ts#L13)
```ts
const atomToPrintable = (atom: Atom<unknown>) =>
  atom.debugLabel \|\| atom.toString();
```

3. Only outputs debugLabel, so not the same logic as [src/utils/useAtomsDevtools.ts]
[src/DevTools/utils/atom-to-printable.ts](https://github.com/jotaijs/jotai-devtools/blob/227170689eb68f006c5e60677d581f4f932f089d/src/DevTools/utils/atom-to-printable.ts#L9)
```ts
export const atomToPrintable = (atom: AnyAtom): string => {
  return atom.debugLabel ? atom.debugLabel : `<unlabeled-${atom}>`;
};
```

...Rest are references to 1, 2, and 3 above.

Reply:
> Good point! There is [one more to update then](https://github.com/jotaijs/jotai-devtools/blob/227170689eb68f006c5e60677d581f4f932f089d/src/utils/useAtomDevtools.ts#L29) 😄

Same concern. Only outputs debugLabel, so not the same logic as [src/utils/useAtomsDevtools.ts](https://github.com/jotaijs/jotai-devtools/blob/227170689eb68f006c5e60677d581f4f932f089d/src/utils/useAtomsDevtools.ts#L11).

Reply:
I agree this logic should be centralized, but can we move centralizing print logic to another PR since it will change the underlying logic?

This PR is to accommodate https://github.com/pmndrs/jotai/pull/2659 in jotai core so that it doesn't break devtools.

Reply:
https://github.com/pmndrs/jotai/pull/2659#discussion_r1677238083

Reply:
You're right! we can do a broader overhaul of how these labels are used in dev tools in other PRs.